ts/helpers/attributesHandler.ts

```diff
@@ -0,0 +1,29 @@
+
+export const getAttributeByPath = (slideAttributes, path) => {
+
+    if(!Array.isArray(path)){
+        throw Error("Invalid path");
+    }
+
+    if(slideAttributes === undefined) {
+        return undefined;
+    }
+
+    for(const node of path){
+        slideAttributes = slideAttributes[0][node];
+        if (slideAttributes === undefined) {
+            return [];
+        }
```

Comment:
Can you let me know what you are trying to do here? I will come around, but it seems strange to reassign the slideAttributes arg and then also return it inside a loop.

Reply:
i hope its clear now after our discussion.